cc/io/VideoCapture.cc

```diff
@@ -27,11 +27,19 @@ NAN_METHOD(VideoCapture::New) {
   FF::TryCatch tryCatch("VideoCapture::New");
   FF_ASSERT_CONSTRUCT_CALL();
   VideoCapture* self = new VideoCapture();
+
+  uint32_t apiPreference = cv::CAP_ANY;
+  if (info.Length() == 2) {
```

Comment:
the latest vertion allow a 3th parameter, so the test should be >=2

and is you want a more cleaner code throw an not implemented yet if more than 2 parameters are provided.

the 3th parameter is only available in openCV ~~ 4.5+

Reply:
New changes should cover all constructor variant a user might try to use and throw if it is not yet implemented